homeassistant/components/comfoconnect/sensor.py

```diff
@@ -72,163 +76,187 @@
 
 _LOGGER = logging.getLogger(__name__)
 
-ATTR_LABEL = "label"
-ATTR_MULTIPLIER = "multiplier"
-ATTR_UNIT = "unit"
+
+@dataclass
+class ComfoconnectRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    sensor_id: int
+
+
+@dataclass
+class ComfoconnectSensorEntityDescription(
+    SensorEntityDescription, ComfoconnectRequiredKeysMixin
+):
+    """Describes Comfoconnect sensor entity."""
+
+    state_class = STATE_CLASS_MEASUREMENT
+    multiplier: float = 1
+
 
 SENSOR_TYPES = {
```

Comment:
`SENSOR_TYPES` can be converted to a `tuple` of `ComfoconnectSensorEntityDescription`. No need to keep the dict keys around anymore as they are now `description.key`.

Reply:
I use them for validation here: https://github.com/home-assistant/core/blob/a21e3aed77fd5d1781101e81ad543271c7173ee9/homeassistant/components/comfoconnect/sensor.py#L237

Is that another way to do this cleanly?

Reply:
You can use a list comprehension to create a key list on the fly. Something like
```py
SENSOR_KEYS = [desc.key for desc in SENSOR_TYPES]
```